demos/nextAuth-demo/.gitignore

```diff
@@ -0,0 +1 @@
+.env.local
```

Comment:
[boulder] There will be other build artifacts and potentially other files that we may want to omit. Running `npx gitignore node` will generate a thorough .gitignore for Node/frontend projects.